### PR TITLE
[IMP] (website_)event: Improve demo data

### PR DESCRIPTION
**Pull request by qmo-odoo**
In order to have an easier, better time showcasing the functionalities of event,
demo data had to be improved.

TaskID: 2170834
PR: #xxxxx

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/event/data/event_demo.xml [hunk 1]
```diff
@@ -61,9 +63,48 @@
...
+        <field name="address_id" ref="event.res_partner_location_0"/>
         <field name="seats_availability">unlimited</field>
         <field name="stage_id" ref="event_stage_booked"/>
+        <field name="kanban_state">blocked</field>
+    </record>
+
+    <record id="message_event_1_0" model="mail.message">
+        <field name="model">event.event</field>
+        <field name="res_id" ref="event.event_1"/>
+        <field name="body"><![CDATA[<p>Hello Marc Demo,<br/>
```
Comment (tde-banana-odoo) [inline addons/event/data/event_demo.xml]: Do not use Cdata, use type="html" instead. That way you have a real syntax highlight in your linter :) .

File: addons/event/data/res_partner_demo.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+
+    <!-- Res Partner that will be used as locations for events -->
+    <record id="res_partner_location_0" model="res.partner">
```
Comment (tde-banana-odoo) [inline addons/event/data/res_partner_demo.xml]: Use already existing partners (maybe tweak one of them if necessary). We have enough demo, no need to pile them up.

File: addons/event/tests/test_event_security.py [hunk 3]
```diff
@@ -86,8 +86,8 @@ def test_event_access_event_manager(self):
         registration = self.env['event.registration'].create({'event_id': event.id, 'name': 'Myself'})
         registration.write({'name': 'Myself2'})
 
-        stage = self.env.ref('event.event_stage_done')
-        stage.write({'name': 'ManagerEnded'})
+        stage = self.env.ref('event.event_stage_cancelled')
```
Comment (tde-banana-odoo) [inline addons/event/tests/test_event_security.py]: Why this change ?

File: addons/website_event/data/event_demo.xml [hunk 4]
```diff
@@ -43,6 +43,19 @@
         <field name="website_published">True</field>
         <field name="subtitle">The Great Reno Balloon Race is the world's largest free hot-air ballooning event.</field>
         <field name="cover_properties">{"background-image": "url('/website_event/static/src/img/event_cover_1.jpg')", "resize_class": "o_record_has_cover o_half_screen_height", "opacity": "0.4"}</field>
+        <field name="description"><![CDATA[
```
Comment (tde-banana-odoo) [inline addons/website_event/data/event_demo.xml]: Same here, do not use cdata.

File: addons/event/tests/test_event_security.py (hunk 3 above)
Comment (qmo-odoo) [reply addons/event/tests/test_event_security.py]: The problem was that, due to the fact that I had to add an event in the "done" stage, this test would break when trying to unlink the stage as it would still be referenced by said event :/. 

To prevent this, a cleaner solution would be to create a stage in the test and delete it to avoid relying on data and stumble on the same kind of problem further down the line.

File: addons/event/data/event_demo.xml [hunk 5]
```diff
@@ -67,8 +69,47 @@
...
+        <field name="parent_id" ref="message_event_1_0"/>
+        <field name="body" type="html"><p>Hi Mitchell Admin,<br/>I will take care of it today !</p></field>
+        <field name="message_type">comment</field>
+        <field name="author_id" ref="base.partner_demo"/>
+    </record>
+    <record id="message_event_1_2" model="mail.message">
+        <field name="model">event.event</field>
+        <field name="res_id" ref="event.event_1"/>
+        <field name="parent_id" ref="message_event_1_1"/>
+        <field name="body" type="html"><p>Great ! The task will stay "blocked" until it is fixed.<br/>
```
Comment (tde-banana-odoo) [inline addons/event/data/event_demo.xml]: The "task" ? Which task ?

File: addons/event/data/event_demo.xml [hunk 6]
```diff
@@ -67,8 +69,47 @@
...
+        </field>
+        <field name="message_type">comment</field>
+        <field name="author_id" ref="base.partner_admin"/>
+    </record>
+    <record id="message_event_1_1" model="mail.message">
+        <field name="model">event.event</field>
+        <field name="res_id" ref="event.event_1"/>
+        <field name="parent_id" ref="message_event_1_0"/>
+        <field name="body" type="html"><p>Hi Mitchell Admin,<br/>I will take care of it today !</p></field>
+        <field name="message_type">comment</field>
```
Comment (tde-banana-odoo) [inline addons/event/data/event_demo.xml]: If we want to have full demo data, we can also set subtype (mail.mt_comment for comments, mail.mt_note for note, should probably be sufficient for standard demo).

File: addons/event/data/event_demo.xml (hunk 5 above)
Comment (qmo-odoo) [reply addons/event/data/event_demo.xml]: Scary pavlovian reflex indeed :D

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff
